lib/web_ui/dev/common.dart

```diff
@@ -20,21 +18,23 @@ const double kMaxDiffRateFailure = 0.28 / 100; // 0.28%
 
 abstract class PlatformBinding {
   static PlatformBinding get instance {
-    if (_instance == null) {
+    PlatformBinding? binding = _instance;
+    if (binding == null) {
       if (io.Platform.isLinux) {
-        _instance = _LinuxBinding();
+        binding = _LinuxBinding();
       } else if (io.Platform.isMacOS) {
-        _instance = _MacBinding();
+        binding = _MacBinding();
       } else if (io.Platform.isWindows) {
-        _instance = _WindowsBinding();
+        binding = _WindowsBinding();
       } else {
         throw '${io.Platform.operatingSystem} is not supported';
       }
+      _instance = binding;
     }
-    return _instance;
+    return binding;
   }
```

Comment:
> Not sure what's the best way to get rid of this (!).

I usually use `_instance ??= _createInstance()` to avoid the `!`, like in your alternative suggestion.